### ISSUE TITLE
New AWB voice

### ISSUE THREAD

**Issue by Ali20177** [CSTR-Edinburgh/merlin#124]
Dear all,

I did the following below for a new voice:

1- I have changed the SLT speaker to the AWB speaker
2- I have successfully did phone alignments
3- I have replaced the existing folder "label_phone_align" with new AWB in both duration and acoustic folders.
4- I have also extracted new features by WORLD vocoder and replaced existing folders accordingly.
4- I have also changed the file_id_list_full as well as cmuarctic.data to the new AWB.
5- I have replace the lab folder in "egs\slt_arctic\s1\slt_arctic_full_data" to the new lab folder that I got it from phone alignments.

But after run Merlin as normal, I got this error (for example):
"the number of frames in label and acoustic features are different: 646 vs 346 (arctic_a0043)". I tried remove this wave file but got similar error with other wav name. 

In other words, I have followed every single step in "https://github.com/CSTR-Edinburgh/merlin/tree/master/egs/build_your_own_voice/s1". But the error is the same while it works well with SLT speakers.

Is there anybody tried before with cmu_us_awb_cg speaker.

Appreciate your ideas and advices if I have missed something.

Thanks again and regards,

**Comment by dreamk73** [comment]
In the question file for the AWB speaker, do you have the label_type set to phone_align? And the subphone_feats to None or coarse_coding? 
When you look at the start and end points of the phones, are they rounded to the nearest 5ms?
Is the sampling frequency the same for the AWB sound files?

**Comment by Ali20177** [comment]
Thanks for your reply. The answer for all your questions is YES.
- label_type = phone_align
- subphone_feats = none or coarse_coding
- sampling frequency = 16Khz
- 5ms
But still same error !!!

**Comment by ronanki** [comment]
As I can see from this audio file:
http://festvox.org/cmu_arctic/cmu_arctic/cmu_us_awb_arctic/wav/arctic_a0043.wav

The length is approx. 5 seconds and can be around 3 sec. excluding silence in the beginning and end. That would count to 600 frames. 

From this, I can assert that there is something wrong with acoustic features. Please check your acoustic features. You should have 200 frames per each second.

**Comment by dreamk73** [comment]
Did you ever resolve this problem? I just tried to create an awb merlin voice myself and got it running without much effort.

- I downloaded the cmu_us_awb_arctic tarball. 
- I ran merlin/misc/scripts/vocoder/world/extract_features_for_merlin.sh to get the acoustic feature files
This creates lab files in full and mono directories.
- I ran merlin/misc/scripts/frontend/festival_utt_to_lab/make_labels <data>/awb/lab <data>/awb/utts ${FESTDIR}/examples/dumpfeats 
 This uses the utt files already present in the awb distribution. So no forced alignment needed. 
- I then ran merlin/misc/scripts/frontend/festival_utt_to_lab/utils/normalize_lab_for_merlin.py <data>/awb/lab/full <data>/awb/label_state_align state_align <data>/awb/file_id_list.scp
(I used state alignments, but I think you should be able to do the same with phone alignments)

**Comment by Ali20177** [comment]
Thank you very much for your helpful comment. I'll try to do it and let you know the progress.
Best regards.

**Comment by Ali20177** [comment]
Dear dreamk73,

Could you please upload only your state alignments files for whole AWB voice and share me the link of it?
Because until now, I got mismatch frames number and alignments error.

Appreciate your help in advance.
Best regards

**Comment by ronanki** [comment]
If you pull the latest code, it has awb_arctic voice as well -- with phone_align labels. 

Just type: ./01_setup.sh to see the list of voices. Make sure you change the labels type from state_align to phone_align in global settings before creating conf files.

**Comment by Ali20177** [comment]
Thank you for letting me that the awb voice has now been included in the Merlin updated version.
I'll keep you in touch ...

Regards.
